.github/workflows/merge-pr.yml

```diff
@@ -94,6 +94,10 @@ jobs:
     steps:
       - name: Check Out Repo
         uses: actions/checkout@v3
+      - name: Install Required Packages
+        run: |
+          sudo apt-get update
+          sudo apt install -y protobuf-compiler
```

Comment:
It would be great if you can explain this requirement is come from and why we need it after upgrade

Reply:
Do we need to add to our README docs for local setup or scripts?

Reply:
It's mostly started to happen in our new instances (runners for actions) which are self hosted by us, and those instances were missing protoc library

Reply:
> Just to clarify `protoc` became required only with this upgrade. Our `main` works fine on the new self-hosted runners without it.

Cool let me check what I got in I believe it's the ability to allow external relay node probably talks over protocol buffers

Note: I realized they give you a heads up by such small changes because in .9.32 they have taken step 2 to reach the goal